docs/releases/1.11.txt

```diff
@@ -230,7 +230,13 @@ Templates
 Tests
 ~~~~~
 
-* ...
+* :func:`~django.test.utils.teardown_databases` has been introduced to make it
+  easier to build custom test runners. :func:`~django.test.utils.teardown_databases` was previously only available on
+  the :class:`~django.test.runner.DiscoverRunner` as a method.
+
+  To be consistent with the :func:`~django.test.utils.teardown_databases` and
```

Comment:
This goes under "Features deprecated" not "Minor Features" -- see https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#deprecating-a-feature for other deprecation requirements.